### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
- [DEMO LINK](https://maksym-nemera.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+  </head>
+  <body class="body">
+    <main class="main">
+      <ul class="stopwatch stopwatch--delete-hours"> <!--change '--delete-hours' to see hours -->
+        <li class="stopwatch__line stopwatch__line--hour"></li>
+        <li class="stopwatch__line stopwatch__line--minute"></li>
+        <li class="stopwatch__line stopwatch__line--second"></li>
+        <li class="stopwatch__line stopwatch__line--milisecond"></li>
+        <li class="stopwatch__center"></li>
+      </ul>
```
Comment (ruslanpashkov) [inline src/index.html:19]: i am not sure that you list items here, but overall it's okay

File: src/styles/blocks/stopwatch.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    top: 15%;
+
+    width: 5%;
+    height: 5%;
+    border-radius: 50%;
+
+    background-color: $color-milisecond-and-dot;
+
+    position: relative;
+  }
```
Comment (ruslanpashkov) [inline src/styles/blocks/stopwatch.scss:25]: i can't figure out what it's for, but it looks cool

File: src/styles/utils/normalize.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+}
```
Comment (ruslanpashkov) [inline src/styles/utils/normalize.scss:9]: it's optional, but i think you can refactor it to smth like that:

```suggestion
body,
ul {
  margin: 0;
}

ul {
  padding: 0;
  list-style: none;
}
```

`<li>`'s will inherit the `<ul>`'s list-style and you will not reset unnecessary padding for body

**Review by ruslanpashkov — APPROVED** [review]

File: src/styles/utils/normalize.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
```
Comment (silchyn) [inline src/styles/utils/normalize.scss:7]:
```suggestion
ul {
```

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,22 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="body">
+    <main class="main">
+      <ul class="stopwatch stopwatch--delete-hours"> <!--change '--delete-hours' to see hours -->
```
Comment (silchyn) [inline src/index.html:13]: ?

**Review by silchyn — CHANGES_REQUESTED** [review]
The second hand should be shorter than the minute hand!

File: src/styles/blocks/stopwatch.scss (hunk 2 above)
Comment (NickMitchell61) [reply src/styles/blocks/stopwatch.scss:25]: @ruslanpashkov 
It's like a mark point to see the midday/midnight or start position :)

image

**Comment by NickMitchell61** [comment]
> The second hand should be shorter than the minute hand!

@silchyn 
the second hand is already shorter than the minute hand

**Comment by serentinos** [comment]
Good job!

**Review by IrynaBalandiukh — APPROVED** [review]
Good job !)

File: src/styles/blocks/stopwatch.scss (hunk 2 above)
Comment (silchyn) [reply src/styles/blocks/stopwatch.scss:25]: pretty confusing

**Review by silchyn — APPROVED** [review]
That was a typo; the second hand should be longer than the minute hand, like on real watches

**Pull request closed without merging**